app/packages/core/src/components/Actions/similar/MaxKWarning.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useTheme } from "@fiftyone/components";
+import ClearOutlinedIcon from "@mui/icons-material/ClearOutlined";
+
+type Props = {
+  onClose: () => void;
+  maxK: number;
+};
+
+const MaxKWarning: React.FunctionComponent<Props> = (props) => {
+  const theme = useTheme();
+
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "space-between",
+        border: "1px solid " + theme.divider,
+        padding: "0.2rem",
+      }}
```

Comment:
super nit: I'm personally generally against inline styles because this style object is allocated in every render :p but this is arguably the most inconsequential thing in the world and feel free to ignore it.